src/index.md

```diff
@@ -1437,79 +1519,6 @@ Option 2: _**`hg`**_ `fetch`
 
 In order to push changesets into the parent repository, some additional configuration is required. The following sections describe the operations that will be performed by users with push access.
 
-#### Generating an SSH Key
-
-All pushes require an ssh key which must be installed on the Mercurial server. The `ssh-keygen` command generates an ssh key. The `-b` option overrides the default number of bits for the key. Allow a few minutes to generate a 4096 bit key; a key of at least 2048 bits is recommended. While it's possible to use ssh without a passphrase, this is **strongly** discouraged. Empty or insecure passphrases may be reset using `ssh-keygen -p`; this doesn't change the keys.
-
-    $ ssh-keygen -t rsa -b 4096
-    Enter file in which to save the key(/u/iris/.ssh/id_rsa):
-    Generating public/private rsa key pair.
-    Enter passphrase(empty for no passphrase):
-    Enter same passphrase again:
-    Your identification has been saved in /u/iris/.ssh/id_rsa.
-    Your public key has been saved in /u/iris/.ssh/id_rsa.pub.
-    The key fingerprint is:
-    md5 4096 c2:b7:00:e6:4b:da:ea:ec:32:30:f5:bd:12:26:04:83 iris@duke
-    The key's randomart image is:
-    +--[ RSA 4096]----+
-    |    E.=          |
-    |     . *         |
-    |      o .   .    |
-    |         + o     |
-    |        S + .    |
-    |       .   + .   |
-    |        + + +..  |
-    |       * . oo+.  |
-    |      o . .o..   |
-    +-----------------+
-
-The `~/.ssh/id_rsa.pub` is a text file containing the public ssh key. This file should be mailed as an attachment along with the JDK username to [keys(at)openjdk.java.net](mailto:keys-at-openjdk.java.net). An administrator will install your key on the server and notify you on completion. This process may take a couple of days.
```

Comment:
I'm not sure how to handle this.  All of the new section is about using Git; however, these instructions for sending to keys@ojn are still needed for the very few Projects which still develop using hg.ojn or for those intending to use cr.ojn.  I referenced them in the past few weeks when helping a user so I'm reluctant to completely eliminate these few sentences quite yet.

Reply:
Can we keep a section "How to do development on the legacy Mercurial server"?

Reply:
I created a "Working with the legacy Mercurial servers" section and pulled all the mercurial stuff from "Repositories" and "Producing a Changeset" there. Some sentences was placed in other parts of the guide where I felt they were more in context. This made the change significantly larger but I also think the result is a lot better and made the guide as a whole more up to date.